content/about/privacy.md

```diff
@@ -0,0 +1,96 @@
+---
+title: "Privacy Policy"
+draft: false
+---
+InnerSource Commons is an independent non-for-profit organisation that creates and shares knowledge about InnerSource for its members. 
+
+This privacy policy will explain how our organisation uses the personal data we collect from you when you use our website and contribute to our projects.
+
+## What Data Do We Collect And Why?
+
+You directly provide InnerSource Commons with most of the data we collect. We collect and process data when you:
+
+- Register to our emailing list: InnerSource Commons collects personal identification information (Name and Email Address) to email you with news, events and articles we think you might like.
+
+- Register to our events: InnerSource Commons collects information that you provide us when you register for one of our events, such as conferences and when you choose to participate in an event as an attendee, speaker, or sponsor. This information includes personal data such as your name, company, and contact information.
+
+- Join our Slack community: When you join the conversation on Slack we store the information you contribute and your conversations.
+
+- Voluntarily contribute to our GitHub Repositories by providing feedback, content or committing code: When you contribute source code, documentation, or other materials to one of our projects (whether on your own behalf or on behalf of your employer), we store the information and content that you contribute. This can include the contributed materials, information required to confirm the provenance of the contribution, and any associated intellectual property. We ask you not to upload personal data to our projects.
+
+- Voluntarily answer questionnaires and surveys: From time to time we may send you questionnaires or surveys for collecting demographic information or assessing your interests and needs, to help us to improve our services. Any such questionnaires or surveys will be entirely voluntary. The purpose and intended use of the information being collected will be explained in the survey itself.
+
+### Information Automatically Collected From You
+
+We also collect certain information automatically. Like many websites, the InnerSource Commons  website uses “cookies” and we obtain certain types of information when your web browser accesses our site. On the InnerSource Commons website Cookies are used for the following purposes and using the following services:
+ 
+Analytics
+
+- Google Analytics - Personal Data: Cookies and Usage Data. We track number of visitors per page on our website.
```

Comment:
Currently we use GA in a "cookieless" way: https://helgeklein.com/blog/2020/06/google-analytics-cookieless-tracking-without-gdpr-consent/

Reply:
Hi Dmitrii, when I click on the lock before the URL I get the bellow, which suggests to me that we do have Google cookies on our website.

![Screenshot 2021-03-03 at 14 24 43](https://user-images.githubusercontent.com/78749459/109819889-69c73400-7c2c-11eb-8ac9-f754a1e72922.png)

Reply:
Seems this happens only on the pages where we have some 3-rd party widgets. Check the cookies on the "clean" page: https://innersourcecommons.net/learn/ :)

Reply:
I know but it is still there. We can't say that we don't use them when they show up. 
@claredillon had a different idea around this. To just say that we use cookies but not list them and point people to their Browser's instructions on how to limit them. 
What do you think?

Reply:
I think it's a good idea and we should keep that in mind – one day we should create a cookie consent.

Reply:
The only thing I am concerned about is that the regulations require us to list the cookies we use and how we use them.

Reply:
I not sure, but can we just say that our widgets from YouTube and Twitter use cookies as required by their terms of usage?

Reply:
We are saying that already bellow. Do you think we should just take out the Analytics part altogether and leave those only?

Reply:
Our google analytics script does not use any cookie :))

Reply:
I deleted the Analytics reference